src/audio/AudioStream.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @author Mugen87 / https://github.com/Mugen87
+ */
+
+function AudioStream( listener ) {
+
+	this.context = listener.context;
+
+	this.gain = this.context.createGain();
+	this.gain.connect( listener.getInput() );
+
+}
+
+Object.assign( AudioStream.prototype, {
+
+	getOutput: function () {
+
+		return this.gain;
+
+	},
+
+	setAudioElement: function ( audioElement ) {
```

Comment:
The problem is that methods like `.play()`, `.pause()`, `.getCurrentTime()` or `.getLoop()` only work if they operate on `. audioElement` (the actual `HTMLAudioElement`). The methods of `Audio` seems to work exclusively with  `AudioBufferSourceNode`.